documentation/Spring-native-vs-Quarkus.asciidoc

```diff
@@ -0,0 +1,104 @@
+== Spring Native vs Quarkus
+
+Nowadays it is very common to write an application and deploy it to
+cloud. Serverless computing and Function-as-a-Service (FaaS) have become
+very popular. While there are a lot of challenges to deploy a Java
+application into the latest cloud environment, the biggest challenges
+developers are facing is memory footprint and the startup time required
+for the Java application, as more of these keeps the host costs high in
+public clouds and Kubernetes clusters. With the introduction of
+frameworks like micronaut and microprofile java processes are getting
+faster and more lightweight. In a similar context Spring has introduced
+https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#overview[Spring Native] which aims to solve the big memory footprint of Spring and its slow startup time to potentially rival the new framework called https://quarkus.io[Quarkus] from Red Hat. This document shortly
+discusses both of these two frameworks and their potential suitabilities
+with devonfw.
+
+=== Quarkus
+
+Quarkus is a full-stack, Kubernetes-native Java framework made for JVMs.
+With its container-first-philosophy and its native compilation Quarkus
+optimizes Java for containers with low memory usage and fast startup
+times.
+
+Quarkus achieves this by the following ways:
+
+* *First Class Support for GraalVM*
+* *Build Time Metadata Processing:* As much processing as possible is
+done at build time, so your application will only contain the classes
+that are actually needed at runtime. This results in less memory usage,
+and also faster startup time as all metadata processing has already been
+done.
+* *Reduction in Reflection Usage:* As much as possible Quarkus
+tries to avoid reflection, reducing startup time and memory usage.
+* *Native Image Pre Boot:* When running in a native image Quarkus
+pre-boots as much of the framework as possible during the native image
+build process. This means that the resulting native image has already
+run most of the startup code and serialized the result into the
+executable, resulting in an even faster startup-time.
+
+This gives Quarkus the potential for a great platform for serverless,
+cloud and Kubernetes environments. For more information about Quarkus
+and its support for devonfw please refer to the link:quarkus.asciidoc[Quarkus introduction guide].
+
+=== Spring Native
+
+*Note: Spring Native is still in Beta. The current version of Spring
```

Comment:
I think the asciidoc Note Block would fit better than just writing it bold.
    ====
    [Note]
    Spring Native is still in Beta. The current version of Spring Native 0.10.5 is designed to be used with 
    Spring Boot 2.5.6.
    ====